assets/css/main.css

```diff
@@ -0,0 +1,105 @@
+@import "@gouvfr/dsfr/dist/dsfr.css";
+@import "@gouvfr/dsfr/dist/utility/utility.css";
+@import "@datagouv/components/dist/style.css";
+
+@import "tailwindcss" important;
```

Comment:
This change is adding a `!important` to the rule :

```
#__nuxt,
#__nuxt > div {
  @apply h-full;
}
```

This is making the `nuxt-loading-indicator` full page instead of 3px.

Reply:
Fixed by not using Tailwind for this